source/game_sa/CTagManager.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+#include "PluginBase.h"
+
+struct tTagDesc {
+    CEntity* m_pEntity;
+    unsigned char m_nAlpha;
+    unsigned char _pad[3];
+};
+
+class CTagManager {
+public:
+    static constexpr unsigned char ucAlphaTagged = 228;
+    static constexpr unsigned int uiTagsNum = 150;
+    static tTagDesc* ms_tagDesc; //ms_tagDesc[150]
```

Comment:
```suggestion
    static tTagDesc(&ms_tagDesc)[MAX_TAGS];
```
use const or contexpr for MAX_TAGS and set it to 150.

Reply:
Fixed